tests/ci_tests/code_formatter/crewai/test_crewai_tasks.py

```diff
@@ -0,0 +1,200 @@
+"""Tests for coffee_maker/code_formatter/crewai/tasks.py"""
+
+from unittest import mock
+
+import pytest
+from crewai import Agent, Task
+
+from coffee_maker.code_formatter.crewai.tasks import (
+    EXPLANATIONS_DELIMITER_END,
+    EXPLANATIONS_DELIMITER_START,
+    MODIFIED_CODE_DELIMITER_END,
+    MODIFIED_CODE_DELIMITER_START,
+    create_reformat_task,
+    create_review_task,
+)
+
```

Comment:
Line 12-16: Added a docstring to the function as specified in the style guide.
```suggestion
    """Copies a file from source to destination.\n\n    Args:\n        source_path (str): The path to the source file.\n        destination_path (str): The path to the destination file.\n    """
```